package.json

```diff
@@ -38,6 +39,9 @@
     "react-dom": "^0.14.7",
     "react-dropzone": "^3.3.2",
     "react-redux": "^4.2.1",
-    "react-select": "^0.9.1"
+    "react-select": "^0.9.1",
+    "url-loader": "^0.5.7",
+    "img-loader": "^1.2.0",
+    "file-loader": "^0.8.4"
```

Comment:
Again, it seems leftover of the investigation which I was doing for bundling the assets for nav bar component. I would check if it works without it and remove it if not necessary.